data/seeds/testFiles.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
You also forgot to use the knex-cleaner library, so you dont have a way to remove the records from your db once you want to run the seedings again, so it will generate duplicates and this is bad for data consistency and performance, we dont want duplicates